osu.Game/Overlays/Comments/CommentsContainer.cs

```diff
@@ -102,7 +102,7 @@ private void load(OverlayColourProvider colourProvider)
                             Padding = new MarginPadding { Horizontal = WaveOverlayContainer.HORIZONTAL_PADDING, Vertical = 20 },
                             Children = new Drawable[]
                             {
-                                avatar = new UpdateableAvatar(api.LocalUser.Value)
+                                avatar = new UpdateableAvatar(api.LocalUser.Value, isInteractive: false)
```

Comment:
value or `isInteractive` has changed, intended?

Reply:
Yes, it should be the same as web behavior, else change web behavior.